src/runtime/contrib/dnnl/dnnl_json_runtime.cc

```diff
@@ -241,6 +243,9 @@ class DNNLJSONRuntime : public JSONRuntimeBase {
           Deconvolution(nid);
         } else if (std::regex_match(op_name, conv_pat)) {
           Convolution(nid);
+        } else if (std::regex_match(op_name, dense_pack_pat) ||
+                   std::regex_match(op_name, stock_dense_pack_pat)) {
+          Dense(nid, true);
```

Comment:
No need in dense version with name "packeddense". That is absolute equivalent of original "Dense" primitive with one additional attribute `weight_layout`. So you may use previous implementation of Dense with taking into account weight layout. Example is below:

```cpp
// yes, GetNodeAttr() supports default value as optional argument 
auto wgh_layout = GetNodeAttr<std::string>(node, "weight_layout", {"NC"});
wgh_tr = wgh_tr.TreatAs(wgh_layout, "NC");
```

Reply:
Fix it. Thanks!